web/datapipeline/sap_systems_projector.go

```diff
@@ -0,0 +1,81 @@
+package datapipeline
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/trento/internal/sapsystem"
+	"github.com/trento-project/trento/web/entities"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+const SAPSystemsApplication = "application"
```

Comment:
I think we could unify these names in `internal` and use the same value in the agent and here. We can even change the agent to send a string if it helps.